src/Keboola/DbExtractor/Configuration/ConfigDefinition.php

```diff
@@ -27,139 +26,11 @@ public function getConfigTreeBuilder() : TreeBuilder
                     ->isRequired()
                     ->cannotBeEmpty()
                 ->end()
-                ->arrayNode('db')
-                    ->children()
-                        ->scalarNode('driver')->end()
-                        ->scalarNode('host')->end()
-                        ->scalarNode('port')->end()
-                        ->scalarNode('database')
-                            ->isRequired()
-                            ->cannotBeEmpty()
-                        ->end()
-                        ->scalarNode('user')
-                            ->isRequired()
-                        ->end()
-                        ->scalarNode('password')->end()
-                        ->scalarNode('#password')->end()
-                        ->append($this->addSshNode())
-                    ->end()
-                ->end()
-                ->append($this->addTablesNode())
-            ->end();
-        // @formatter:on
-        return $treeBuilder;
-    }
-
-    public function addSshNode(): ArrayNodeDefinition
-    {
-        $builder = new TreeBuilder();
-
-        /** @var ArrayNodeDefinition */
-        $node = $builder->root('ssh');
-
-        // @formatter:off
-        $node
-            ->children()
-                ->booleanNode('enabled')->end()
-                ->arrayNode('keys')
-                    ->children()
-                        ->scalarNode('private')->end()
-                        ->scalarNode('#private')->end()
-                        ->scalarNode('public')->end()
-                    ->end()
-                ->end()
-                ->scalarNode('sshHost')->end()
-                ->scalarNode('sshPort')->end()
-                ->scalarNode('remoteHost')->end()
-                ->scalarNode('remotePort')->end()
-                ->scalarNode('localPort')->end()
-                ->scalarNode('user')->end()
+                ->append($this->getDbParametersDefinition())
+                ->append($this->getTablesNode())
```

Comment:
Tady je vidět, že se metody stejného principu jmenují různě...

Reply:
Jasne, sjednotim to.